### PR TITLE
port over rake task from Hyku main

### PR DESCRIPTION
**Pull request by ShanaLMoore**
~~a user was unable to export work types that had no files, due to a bug in the solr query. The fix was done in bulkrax. This PR pulls it in by upgrading Bulkrax's version.~~

UPDATE: This PR pulls over a rake task that hooks into bulkrax, which will allow a user to reset data more easily. 

ref: 

# Story

~~Refs https://github.com/scientist-softserv/britishlibrary/issues/289~~

# Expected Behavior Before Changes

~~User received a solr error when exporting works that didn't have filesets, even though they chose metadata only.~~

UPDATE: user was unable to easily reset data in a given environment

# Expected Behavior After Changes

~~A user can successfully export.~~

UPDATE: A user can run a rake task to reset data, in a given env.

### CODE REVIEW

**Review by kirkkwang — APPROVED** [review]

**Comment by jeremyf** [comment]
I checked the failures and they might be legit.  I'm diving in now.

File: lib/tasks/reset.rake [hunk 1]
```diff
@@ -0,0 +1,30 @@
+namespace :reset do
```
Comment (ShanaLMoore) [inline lib/tasks/reset.rake:1]: With the upgrade in bulkrax, a reset task was added. We need this corresponding bulkrax in order to run it.

**Comment by alishaevn** [comment]
I no longer see a bulkrax update in the changed files. was it done elsewhere and now this pr is just for adding the reset task?

**Comment by ShanaLMoore** [comment]
> I no longer see a bulkrax update in the changed files. was it done elsewhere and now this pr is just for adding the reset task?

Yes, by time I got back to this PR it had already been updated elsewhere. We still need this rake task though so I'll update the description. cc @alishaevn

**Pull request merged**